sig-scalability/slos/api_extensions_latency.md

```diff
@@ -0,0 +1,6 @@
+## API call extension points latency SLIs details
+
+### User stories
+- As an administrator, if API calls are slow, I would like to know if this is
+because slow extension points (admission plugins, webhooks, initializers) and
+if so which ones are responsible for it.
```

Comment:
Are we planning to define some SLIs here?

Reply:
There are already SLIs - but definition of SLIs/SLOs are in the slos.md file.

Reply:
I'm already having more the SLI per file, e.g.:
1. one file for both read-only and muting API calls
2. one file for all intiailized, admission plugins and webhooks

I think merging it further is a bad idea, because they pretty much don't share anything.

Reply:
Sorry, I think I'm misunderstanding. I was suggesting above that we move the SLI definitions into the relevant files instead of putting them in the root file - there's no merging involved there.

Reply:
Ping.

Reply:
Sorry - apparently I forgot to send this comment.

I definitely want to have a single table with just all SLIs/SLOs definition (wihtout any comments) - that is what 90% of people would be interested in (and not interested in anything else).

We may consider duplicating the SLI/SLO definition into those files, but personally, I don't think that's a good idea, because that would make people read those dosc instead of the main one.

People should just read the "slos.md", and only if they are intersted in more details, go into those smaller files.

Reply:
> 90% of people would be interested in

Seems like you also want some SLO for our readers :)

And yeah, that sgtm.